_rules/form-control-accessible-name-e086e5.md

```diff
@@ -210,15 +210,15 @@ Hidden to assistive technologies.
 Role has [explicitly](#explicit-role) been set to something that isn't a form field.
 
 ```html
-<input role="presentation" />
+<input role="presentation" disabled />
 ```
 
 #### Inapplicable Example 4
 
 Option inherits from input, but has a required context role of listbox which inherits from select. We should therefore not consider option as applicable.
 
 ```html
-<select role="none">
+<select role="none" disabled>
```

Comment:
This now falls victim to presentational role inheritance so the description isn't quite accurate; it is also inapplicable by virtue of the `<option>` elements being presentational (https://w3c.github.io/aria/#presentation). We might be okay with that though 🤷

Reply:
I think this is OK…

---
First, this is what the example was trying to do previously. If we want to fix the inheritance, this would be for another PR.

---
Second, I am not sure that the inheritance actually happens in this case. Chrome does hide the `select` while giving a role of `menuitem` to the `option`s.

I guess the relevant parts about presentational inheritance are
> When an explicit or inherited role of presentation is applied to an element with the implicit semantic of a WAI-ARIA role that has required owned elements, in addition to the element with the explicit role of presentation, the user agent MUST apply an inherited role of presentation to any owned elements that do not have an explicit role defined.

This is OK because `combobox` has no required owned elements.

> Also, when an explicit or inherited role of presentation is applied to a host language element which has required children as defined by the host language specification, in addition to the element with the explicit role of presentation, the user agent MUST apply an inherited role of presentation to any required children that do not have an explicit role defined.

This is indeed not fully OK because `select` content model is "Zero or more option, optgroup, and script-supporting elements." I am not sure why Chrome doesn't inherit the role, however… Maybe it interprets "**zero** or more" as them not really being needed (since zero is OK) which is a bit weird…

---
Anyway, since giving the `select` a role of `none` was the intend of this example, I say that fixing the presentational inheritance might be topic for another PR, especially since it doesn't change the outcome of the test case (and only makes the description inaccurate).